index.html

```diff
@@ -1030,10 +1030,16 @@
 which is just a set of characteristics that is too small or unstable to be useful
 for following them through time.
 
+It is important to keep in mind that a person's [=identity=] may differ from their
+[=legal identity=]. A [=person=]'s <dfn>legal identity</dfn> is the set of characteristics
+that a governmental or otherwise official entity recognizes as corresponding to that
```

Comment:
Is "Eye color: blue" part of my legal identity, or is my legal identity just the _identifiers_ a government has assigned to me?